src/cider/nrepl/middleware/stacktrace.clj

```diff
@@ -187,6 +188,37 @@
   (when-let [data (ex-data e)]
     (into {} (filter (comp (complement ex-data-blacklist) key) data))))
 
+
+(def spec-abbrev (delay
+                   (if (try (require 'clojure.spec.alpha) true
```

Comment:
Seems to me you should check for the non-alpha version of the ns as well, as it's eventually going to be released.

Reply:
Good idea.